test/fixtures/results/quickstart/nist_iam.json

```diff
@@ -78,5 +78,25 @@
         "Level": "Warning",
         "Message": "Obsolete DependsOn on resource (rSysAdminRole), dependency already enforced by \"!Ref\" at Resources/rSysAdminProfile/Properties/Roles/0/Ref/rSysAdminRole",
         "Filename": "test/templates/quickstart/nist_iam.yaml"
+    },
+    {
+        "Rule": {
+            "Id": "E3067",
+            "Description": "Check for valid IAM Permissions",
+            "ShortDescription": "Check IAM Permission configuration"
+        },
+        "Location": {
+            "Start": {
+                "ColumnNumber": 9,
+                "LineNumber": 163
+            },
+            "End": {
+                "ColumnNumber": 9,
+                "LineNumber": 163
+            }
+        },
+        "Level": "Error",
+        "Message": "Invalid permission \"Get*\" for \"appstream\" found in permissions",
```

Comment:
I'm getting confused a little bit here.  Do errors in IAM policies result in an Error or a failure of the stack being deployed?  Should this be a warning?  I really can't find an appstream Get API call (which if I'm missing one my thought process is completely off so double check my work here).

Reply:
Yes, a warning makes more sense. Invalid IAM policies will just get deployed without errors in CloudFormation. I can turn this into a warning if that makes more sense.